tests/integration/backward_compatible/db_test.py

```diff
@@ -0,0 +1,123 @@
+from typing import List
```

Comment:
I believe if we say that we provide thread safety for module and connection, we should add a test for it

Reply:
Added a test which creates cursors in parallel at: https://github.com/hazelcast/hazelcast-python-client/pull/586/commits/97482355eb0465ec4b0543b5dc1f48df0800db88